public/app/percona/backup/components/BackupInventory/BackupInventory.service.ts

```diff
@@ -32,6 +32,13 @@ export const BackupInventoryService = {
       })
     );
   },
+  async restore(serviceId: string, locationId: string, artifactId: string): Promise<any> {
+    return api.post(`${BASE_URL}/Backups/RestoreBackup`, {
+      service_id: serviceId,
+      location_id: locationId,
+      artifact_id: artifactId,
+    });
+  },
```

Comment:
```suggestion
  async restore(serviceId: string, locationId: string, artifactId: string): Promise<any> => 
    api.post(`${BASE_URL}/Backups/RestoreBackup`, {
      service_id: serviceId,
      location_id: locationId,
      artifact_id: artifactId,
    });
```

Reply:
right, sorry, I missed that, let's keep it as it is for now

Reply:
👍